server/vulnerabilities/macoffice/analyzer.go

```diff
@@ -0,0 +1,187 @@
+package macoffice
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"sort"
+
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/vulnerabilities/io"
+	"github.com/fleetdm/fleet/v4/server/vulnerabilities/utils"
+)
+
+func latestReleaseNotes(vulnPath string) (ReleaseNotes, error) {
```

Comment:
I believe this function generates the release note.
Can we call it something like `generateLatestReleaseNotes` / `makeLatestReleaseNotes` ...?

Reply:
No, this function loads the 'latest' release note from the file system. At any given time there could be more than one release note files on the file system, we want to always use the latest. I can rename it to `getLatestReleaseNotes(..)` if that makes more sense.

Reply:
`getLatestReleaseNotes` sounds better. thx.